### PR TITLE
September 2022 notes and October Stubb

### PR DESCRIPTION
**Pull request by jessemapel**

### CODE REVIEW

File: meetings/2022-09-08.md [hunk 1]
```diff
@@ -4,15 +4,44 @@
...
+  - Has a few members now.
+  - Will start working soon
+  - If there are gaps then we will address those as they come up
+  - Potential names
+    - No consensus yet
+    - Likely will go back to the community with a few options
+    - People do not seem concerned about the exact name as long as it is
+      - concise
+      - distinct
+      - doesn't have bagged
```
Comment (rbeyer) [inline meetings/2022-09-08.md:28]: "baggage" I assume?

File: meetings/2022-10-13.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+-
+
+## Agenda / Notes
+
+- Action Items from last month
+
+- ISIS Name Change Working Group
+
+- Open discussion
+
```
Comment (rbeyer) [inline meetings/2022-10-13.md:16]: Follow up discussion on https://github.com/planetarysoftware/ISIS_TC/issues/167

**Review by rbeyer — CHANGES_REQUESTED** [review]
Typo in notes, and suggested agenda item for Oct.

**Review by rbeyer — APPROVED** [review]

**Comment by jlaura** [comment]
All - I am not sure that I will be able to attend this call tomorrow as it is fall break for the family and I will be out of town. I will make every attempt to join. 

I want to note that there is *still* no discussion on [#5029](https://github.com/USGS-Astrogeology/ISIS3/issues/5029) about concrete issues with GTest conversion and test writing. As I indicated in #167, from my perspective, the generic complaining with no realistically actionable input is counterproductive at best. It is my hope that concrete and actionable discussion occurs on [#5029](https://github.com/USGS-Astrogeology/ISIS3/issues/5029) and/or on some issue in this repository.

**Review by jlaura — APPROVED** [review]

**Pull request merged**